src/stories/templates/searchHead/searchHeadTemplate.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { Story, Meta } from '@storybook/react/types-6-0';
+import SearchHead, {IsearchHeadForm} from '../../../components/templates/searchForm';
+
+
+export default {
+    title: 'Components/Templates/SearchHead',
+    component: SearchHead,
+
+  } as Meta;
+
+  const Template: Story<IsearchHeadForm> = (args:IsearchHeadForm) => <SearchHead {...args} />;
```

Comment:
The spacing here looks weird to me. But this will be automatically solved when we will be using a linter so could you please adjust the spacing manually for now?